tsl/src/nodes/decompress_chunk/planner.c

```diff
@@ -145,6 +146,26 @@ typedef struct
 
 } DecompressionMapContext;
 
+typedef struct VectorQualInfoDecompressChunk
+{
+	VectorQualInfo vqinfo;
+	const UncompressedColumnInfo *colinfo;
+} VectorQualInfoDecompressChunk;
+
+static bool *
+build_vector_attrs_array(const UncompressedColumnInfo *colinfo, const CompressionInfo *info)
+{
+	const unsigned short arrlen = info->chunk_rel->max_attr + 1;
+	bool *vector_attrs = palloc0(sizeof(bool) * arrlen);
```

Comment:
Nit: Since you're setting all values, there is no need to zero the array first.
```suggestion
	bool *vector_attrs = palloc(sizeof(bool) * arrlen);
```

Reply:
Fixed